docs/source/github_config_keys.rst

```diff
@@ -0,0 +1,23 @@
+.. _github_config_keys:
+
+ .. note::
+
+    **git-machete** will try to infer GitHub API server URL from ``git remote``.
```

Comment:
Nit: I don't think bold for `git-machete` is justfiied

Reply:
@PawelLipski While reading, it made it easier for me to instantly realise that it's the app name but i am open to revert to non-bold

Reply:
You can also change to passive voice:
```suggestion
    GitHub API server URL will be inferred from ``git remote``.
```